debian/container_exit.sh

```diff
@@ -4,3 +4,4 @@ LANG=C
 LC_CTYPE=C
 export LANG LC_CTYPE
 apt-get clean
+rm -f /etc/machine-id /var/lib/dbus/machine-id
```

Comment:
This is how I do it on Debian/Ubuntu, there was a reason but I don't recall. This should be tested before pushed.  Not sure if my Debian/Ubuntu virtual cluster still works.
```
echo "uninitialized" > /etc/machine-id && \
rm -v /var/lib/dbus/machine-id
```

Reply:
Booted a Debian:12.0 node and it worked fine.

Reply:
I vote remove